kernel/src/main/java/org/kframework/kprove/KProveOptions.java

```diff
@@ -55,33 +55,40 @@ public synchronized File specFile(FileUtil files) {
     @ParametersDelegate
     public PrintOptions print = new PrintOptions();
 
-    @Parameter(names="--branching-allowed", arity = 1, description="Number of branching events allowed before a forcible stop.")
+    @Parameter(names="--branching-allowed", descriptionKey = "number", arity = 1,
+            description="Number of branching events allowed before a forcible stop.")
     public int branchingAllowed = Integer.MAX_VALUE;
 
-    @Parameter(names={"--spec-module", "-sm"}, description="Name of module containing specification to prove")
+    @Parameter(names={"--spec-module", "-sm"}, descriptionKey = "name",
```

Comment:
Now that you made it `name`, you should change the other instances for kompile and kast as well.